src/Whoops/Util/SystemFacade.php

```diff
@@ -20,23 +20,19 @@ public function startOutputBuffering()
 
     /**
      * @param callable $handler
-     * @param int      $types
+     * @param int      $types By default E_ALL | E_STRICT
      *
-     * @return callable|null
+     * @return string|null
      */
-    public function setErrorHandler(callable $handler, $types = 'use-php-defaults')
+    public function setErrorHandler(callable $handler, $types = null)
     {
-        // Workaround for PHP 5.5
-        if ($types === 'use-php-defaults') {
-            $types = E_ALL | E_STRICT;
-        }
-        return set_error_handler($handler, $types);
+        return set_error_handler($handler, $types === null ? E_ALL | E_STRICT : $types);
     }
 
     /**
      * @param callable $handler
      *
-     * @return callable|null
+     * @return string|null
```

Comment:
This is not true. The function implementation returns the return value of PHP built-in `set_exception_handler`. [The documentation](http://php.net/manual/en/function.set-exception-handler.php) states it returns a callable. Something is wrong here, and the proposed change needs much more explanation to be included.